cmd/deploy/deploy.go

```diff
@@ -297,7 +297,7 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		Name:       deployOptions.Name,
 		Namespace:  deployOptions.Manifest.Namespace,
 		Repository: os.Getenv(model.GithubRepositoryEnvVar),
```

Comment:
moving this to constants would make sense?

Reply:
It makes sense but I would avoid doing it in this PR to keep it as isolated as possible. I'd create an issue to move all the model constants to its own package later 😄